umh-core/examples/example-config-redpanda.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2025 UMH Systems GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+agent:
+  metricsPort: 8080
+
+redpanda:
+  name: redpanda
+  desiredState: active
+  redpandaServiceConfig:
```

Comment:
do we have a plan on how this redpandaServiceConfig will behave in the future? so an outlook into what config options we will provide and how they will be structured?

Reply:
no, tbh i dont think there will be much to configure here.

Reply:
but if we add some, i would expose it in the same structure that redpanda uses internally